CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## unreleased
 * Remove `iosBaseSDK`, `iosDeploymentTarget`, `iosIdentifierForVendor`, `deviceAppGeneratedPersistentUuid`, and `deviceScreenOrientation` from `BTAnalyticsMetadata`
+* Breaking Changes
+  * BraintreePaymentFlow
+    * Replaced deprecated `SFAuthenticationSession` with `ASWebAuthenticationSession` in payment flow
+    * Removed `BTViewControllerPresentingDelegate` protocol and correlating methods
+    * Removed `load` method from `BTPaymentFlowClient`
+    * Removed `onPaymentCancel` and `returnURLScheme` methods from `BTPaymentFlowClientDelegate`
+    * Removed `canHandleAppSwitchReturnURL` method from `BTPaymentFlowRequestDelegate`
+    * Removed `viewControllerPresentingDelegate` property from `BTPaymentFlowClient`
```

Comment:
More CHANGELOG nit picks - do we need to add these, if they aren't intended for merchant use? Good opportunity to add no-docs to these?

Also another nitpick -  if we did end up keeping these, we can reduce some wordiness by doing syntax like "Remove `BTPaymentFlowRequestDelegate.canHandleAppSwitchReturn(_:)`"

Reply:
That's a good call - I think removing them and adding `nodoc` to the delegates and methods is a good call. Probably not worth calling out in the changelog as it may confuse merchants as to what they were/were not supposed to be using in the flow.

Reply:
Updated here: https://github.com/braintree/braintree_ios/pull/952/commits/c78e3af922d931f8a56e5cce9a057f77b11eb9ad